mapit/shortcuts.py

```diff
@@ -50,25 +50,24 @@ def output_json(out, code=200):
             out['debug_db_queries'] = connection.queries
         indent = 4
     encoder = GEOS_JSONEncoder(ensure_ascii=False, indent=indent)
```

Comment:
This encoder is now only used in the new else so can be moved to inside it.

Reply:
Perhaps could have a `json_dumps_params = {'ensure_ascii': False, 'indent': indent}` here, then can use that in both paths.